client/my-sites/email/email-providers-stacked-comparison/index.tsx

```diff
@@ -143,6 +158,26 @@ const EmailProvidersStackedComparison: FunctionComponent< EmailProvidersStackedC
 					onExpandedChange={ onExpandedStateChange }
 				/>
 			) }
+
+			{ showEmailForwardLink && selectedSite && (
+				<div className="email-providers-stacked-comparison__email-forward-section">
+					{ translate( 'Looking for a free email solution?' ) }{ ' ' }
+					{ translate( 'Start with {{link}}Email Forwarding{{/link}}.', {
```

Comment:
I suspect it's too late but next time I think it may be better to merge strings like this (that are short and closely relate to each other) - this should ease the work of translators, and probably lead to better translations.